jscatter/label_placement/center_of_mass.py

```diff
@@ -0,0 +1,105 @@
+import numpy as np
+import numpy.typing as npt
+from numba import njit
```

Comment:
Does this mean numba is now a dependency of jscatter? I could imagine that would make it challenging to use on newer versions of Python.

Reply:
Yeah, numbs is a dependency now, which already led to issues with Python v3.13 for which I had to drop support because Numba doesn't yet support it. I might do a performance test against the plain Python version to see if the jit compiled methods actually boost the performance noticeably.